exchangeSetService_OpenApi_definition.yaml

```diff
@@ -205,6 +205,70 @@ paths:
         fulfilmentResponse:
           $ref: "#/components/callbacks/fulfilmentResponse"
 
+  /auth/client_credentials:
+    post:
+      summary: Get token from AAD.
+      operationId: getESSTokenUsingClientCredentials
+      description: |
+        Returns a token direct from AAD using Client Credentials
+      requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  client_id:
+                    type: string
+                    description: The application ID that's assigned to the distributor's app. This will be provided to the distributor when they register their application
```

Comment:
Really minor point but we should have a full stop at the end of the description.

Reply:
fixed